active/InformationBar/InformationBar.md

```diff
@@ -0,0 +1,536 @@
+
+## Add buttons
+By default, an 'X' close button will appear as the right most component in the bar.
+
+An additional action button can be added by setting the ActionButtonContent and ActionButtonCommand properties. There is also built-in support for a single hyperlink button to ensure text contrast remains accessible with the various background colors. We recommend that only a single action button or hyperlink button is set. Outside of these, additional action buttons and hyperlinks can be added via custom content.
+
+XAML
+```xml
+<StackPanel x:Name="ContentArea">
+    <controls:InfoBar x:Name="UnsuccessfulSaveNotification"
+        Severity="Warning"
+        Title="Error while saving"
+        Message="Your document was unable to be saved."
+        ActionButtonContent="Try again"
+        ActionButtonCommand="SaveDocuments">
+    </controls:InfoBar>
+</StackPanel>
+```
+
+![A mockup of an InfoBar with a single line message and an action button](images/Warning_Button.png)
+
+XAML
+```xml
+<StackPanel x:Name="ContentArea">
+    <controls:InfoBar x:Name="ConnectionErrorNotification"
+        Title="Error while saving"
+        Message="Lorem ipsum long message">
+        <controls:InfoBar.HyperlinkButtonContent>
+            <HyperlinkButton
+                Content="www.microsoft.com" 
+                NavigateUri="http://www.microsoft.com"/>
+        <controls:InfoBar.HyperlinkButtonContent/>
+    </controls:InfoBar>
+</StackPanel>
+```
+![A mockup of an InfoBar with a single line message and hyperlink](images/Default_MessageHyperlink.png)
+![A mockup of an InfoBar with a message expanding multiple lines and a hyperlink](images/Docked_MultiLineIconTitleHyperlink.png)
+
+## Custom content
+Content can be added to an InfoBar using the Content property. It will appear below the message and above the action or hyperlink buttons if defined. The InfoBar will expand to fit the content defined.
+
+XAML
+```xml
+<StackPanel x:Name="ContentArea">
+    <controls:InfoBar x:Name="TermsAndConditionsNotification"
+        Title="Update Complete!"  
+        Message="Before you can proceed you must read and accept the new Terms and Conditions.">
+        <controls:InfoBar.HyperlinkButtonContent>
+            <HyperlinkButton Content="Read here" NavigateUri="www.microsoft.com"/>
+        </controls:InfoBar.HyperlinkButtonContent>
+    </controls:InfoBar>
+</StackPanel>
+```
+
+![A sketch of an InfoBar in its default state with a hyperlink and message](images/Information_CustomContent.jpg)
+Mockup with custom content to be added.
+
+## Content wrapping
+By default, the text set in the Message property will wrap vertically in the control underneath the other visual components.
+
+XAML
+```xml
+<StackPanel x:Name="ContentArea">
```

Comment:
When the info bar is closed, is there an animation? Or a recommended way for me to add an animation?

Reply:
Currently, no.  I think by default no animations should be shown for these inline InfoBars however if a developer wanted to include their own -- e.g. coming down from the closest orientation (top, bottom, left, right) -- then I think that should be supported. Via storyboards? @teaP?

Reply:
The text in the Canceling example teases the possibility of an animation. Do we need to give an example of how to do one?

Reply:
Removed mention of animation 😊